habitat-lab/habitat/datasets/rearrange/samplers/receptacle.py

```diff
@@ -15,11 +15,12 @@
 import corrade as cr
 import magnum as mn
 import numpy as np
-import trimesh
```

Comment:
@ASzot removing trimesh, so CI on SIRo should pass now (once we can allocate an image).

Reply:
CI issues resolved and test are running